javascript/src/main/components/TutorAssignment/TutorAssignmentSelect.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
+import { Cascader } from "antd";
+import { asHumanQuarter } from "main/utils/quarter";
+
+import {
+  quarterProvider as defaultQuarterProvider,
+  courseProvider as defaultCourseProvider,
+  tutorAssignmentProvider as defaultTutorAssignmentProvider
+} from "main/services/selectorSupport.js"
+
+export const selectorIndices = {
+  QUARTER: 0,
+  COURSE: 1,
+  TUTOR_ASSIGNMENT: 2
+};
+
+export default function TutorAssignmentSelect(props) {
+  const {
+    quarterProvider = defaultQuarterProvider,
+    courseProvider = defaultCourseProvider,
+    tutorAssignmentProvider = defaultTutorAssignmentProvider,
+    onChange,
+    ...otherProps
+  } = props;
+  const [options, setOptions] = useState(null);
+
+  const onCascaderChange = (_value) => {
+    onChange(selectorIndices.TUTOR_ASSIGNMENT);
+  };
```

Comment:
This should be `value[2]` so it actually returns the tutor assignment. The tests pass because I believe the tutor assignment id used for testing also happens to be `2`.
```suggestion
  const onCascaderChange = (value) => {
    onChange(value[selectorIndices.TUTOR_ASSIGNMENT]);
  };
```